packages/perseus-editor/src/components/graph-locked-figures/locked-function-settings.tsx

```diff
@@ -0,0 +1,246 @@
+        ]);
+    }, [domain]);
+
+    // Generic function for handling property changes (except for 'domain')
+    function handlePropChange(property: string, newValue: string) {
+        const updatedProps: Partial<LockedFunctionType> = {};
+        updatedProps[property] = newValue;
+        onChangeProps(updatedProps);
+    }
+
+    /*
+     Reason for having a separate 'propChange' function for 'domain':
+        Domain entries are optional. Their default value is +/- Infinity.
+        Since input fields that are empty evaluate to zero, there needs to be
+            dedicated code to convert empty to Infinity.
+     */
+    function handleDomainChange(limitIndex: number, newValueString: string) {
+        const newDomainEntries = [...domainEntries];
+        newDomainEntries[limitIndex] = newValueString;
+        setDomainEntries(newDomainEntries);
+        const newDomain: Interval | undefined = [
+            domainLimits[0],
+            domainLimits[1],
+        ];
+
+        let newValue = parseFloat(newValueString);
+        if (newValueString === "" && limitIndex === 0) {
+            newValue = -Infinity;
+        } else if (newValueString === "" && limitIndex === 1) {
+            newValue = Infinity;
+        }
+        newDomain[limitIndex] = newValue;
+        onChangeProps({domain: newDomain});
+    }
+
+    return (
+        <UniqueIDProvider
+            mockOnFirstRender={true}
+            scope="locked-function-settings"
+        >
+            {(ids) => (
+                <PerseusEditorAccordion
+                    expanded={props.expanded}
+                    onToggle={props.onToggle}
+                    header={
+                        <View style={styles.row}>
+                            <LabelLarge style={styles.accordionHeader}>
+                                {lineLabel}
+                            </LabelLarge>
+                            <Strut size={spacing.xSmall_8} />
+                            <LineSwatch
+                                color={lineColor}
+                                lineStyle={strokeStyle}
+                            />
+                        </View>
+                    }
+                >
+                    <View style={[styles.row, styles.spaceUnder]}>
+                        {/* Line color settings */}
+                        <ColorSelect
+                            selectedValue={lineColor}
+                            onChange={handlePropChange.bind(null, "color")}
+                        />
+                        <Strut size={spacing.small_12} />
+
+                        {/* Line style settings */}
+                        <LineStrokeSelect
+                            selectedValue={strokeStyle}
+                            onChange={handlePropChange.bind(
+                                null,
+                                "strokeStyle",
+                            )}
+                        />
+                    </View>
+
```

Comment:
Nit: Add a comment here to stay consistent with the other comments.

```
{/* Directional axis settings */}
```

Reply:
Added.